java-cfenv/src/main/java/io/pivotal/cfenv/core/CfEnv.java

```diff
@@ -197,6 +197,21 @@ public CfCredentials findCredentialsByTag(String... spec) {
 		return cfService.getCredentials();
 	}
 
+	public List<CfVolume> findVolumesByName(String... spec) {
```

Comment:
@scottfrederick commented to me out of band that we could do without these finer methods and instead keep with `.findServiceByName("blah").getVolumes()`, which seems reasonable to me as that doesn't seem onerous and helps to keep the api surface smaller.

Reply:
Sure.  Smaller footprint is probably better.